code/controllers/subsystem/dbcore.dm

```diff
@@ -264,26 +264,39 @@ SUBSYSTEM_DEF(dbcore)
   * * querys - List of queries to execute
   * * warn - Boolean to warn on query failure
   * * qdel - Boolean to enable auto qdel of queries
+  * * assoc - Boolean to enable support for an associative list of queries
   */
-/datum/controller/subsystem/dbcore/proc/MassExecute(list/querys, warn = FALSE, qdel = FALSE)
+/datum/controller/subsystem/dbcore/proc/MassExecute(list/querys, warn = FALSE, qdel = FALSE, assoc = FALSE)
```

Comment:
The correct plural is `queries`. Why does tg sin like this

Reply:
You tell me